### PR TITLE
add watchify to speed up builds

### PR DESCRIPTION
**Pull request by mistermendez**

### CODE REVIEW

**Comment by ksunika** [comment]
Looks fine to me, if @chibzu agrees, we can merge.

**Comment by chibzu** [comment]
Are there changes to the dev workflow we should explain in README.md?

**Comment by mistermendez** [comment]
YES, good catch

**Comment by mistermendez** [comment]
@chibzu README updated

File: README.md [hunk 1]
```diff
@@ -47,48 +47,38 @@ This simple test HTML page can also be hosted on your environment to showcase ht
 ```
 
 ## Developer Setup
-This section provides step by step guide on setting up this repo and create simple local hosting.
-You need [npm](https://www.npmjs.org/) installed on your computer.
-From the root project directory run these commands from the command line:
+1. Install [brew](http://brew.sh):
```
Comment (chibzu) [inline README.md]: Brew is an OSX specific dependency manager. This ties our setup instructions specifically to OSX, which I disagree with. We shouldn't be giving instructions that only work on one of three major OS options.

**Comment by mistermendez** [comment]
@chibzu more generic instructions

**Comment by chibzu** [comment]
Looks good.

**Pull request merged**